autoload/black.vim

```diff
@@ -98,13 +98,47 @@ if _initialize_black_env():
   import black
   import time
 
-def Black():
+def get_target_version(tv):
```

Comment:
Should we let Black (or the script) raise on incorrect versions instead of simply filtering? Could be more transparent to users. Also, I'm not entirely sure if this is what's happening here but anyways. Making sure the version is supported by Black is a nice touch though 👌

Reply:
I'll take a closer look at it in the evening. I wrote the code ages ago, so I don't remember without looking at the code.

Reply:
The function is needed because the `target_versions` argument in `black.FileMode` expects a set of the black.TargetVersion` values, so this function maps the string passed by the user to the corresponding `black.TargetVersion` value. You are right, being more transparent to the user is a good thing, hence I decided to print a warning that the default target version is used when Black does not recognize the passed target version.

Reply:
Yep, that seems reasonable! Feel free to mark as resolved